terraform/modules/gost_website/variables.tf

```
@@ -41,6 +41,15 @@ variable "logs_bucket_versioning" {
   default     = true
 }
 
+variable "managed_waf_rules" {
+  description = "List of parameters for managed WAF rules"
+  type = list(object({
+    name              = string
+    priority          = string
+    metric_visibility = bool
+  }))
```

This is admittedly a trivial nit, but since the name of each rule needs (I think?) to be unique, it might make sense to implement this variable as a `map` type keyed by the `name`, which would ensure that the same rule name could only be used once. 

```suggestion
  type = map(object({
    priority          = string
    metric_visibility = bool
  }))
```

---

I actually like this idea better than what I originally had. Much cleaner and easier to understand.